frontend/src/css/modules/sidebar.scss

```diff
@@ -28,32 +29,36 @@
     }
     li {
         a {
```

Comment:
## Nesting Required 
 Nesting the properties like
a {
            &:hover {}
 } 
will be better here :)